.eslint-doc-generatorrc.js

```diff
@@ -1,9 +1,7 @@
-'use strict';
-
-const prettier = require('prettier');
+import prettier from 'prettier';
 
 /** @type {import('eslint-doc-generator').GenerateOptions} */
-module.exports = {
+export default {
```

Comment:
I believe we need to update this for the jsdoc comment to work properly:

```suggestion
const config = {
```

and below:

```js
export default config;
```